docs/sdk-beta/getting-started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## Pre-requisites
+
+1. [Create an account](https://www.dronedeploy.com/signup.html) on DroneDeploy if you do not have one already.
+1. Become a [DroneDeploy Developer](../developer-mode/developer.md)
+1. Request an API key by contacting [developer@dronedeploy.com](developer@dronedeploy.com) if you do not already have one.
+
+## Sample App
+
+To get you started, we have created a sample app. By deploying this sample app, you will learn to use every aspect of the DroneDeploy SDK.
+
+### Overview
+
+The app you will be deploying will enable the following flow:
+1. A user creates an export
+1. When the export completes, an export complete Trigger will go off
```

Comment:
Trigger is capitalized here, and not on the line below.  I'd say make them the same for consistency.